subgraphs/mux-protocol/src/entities/account.ts

```diff
@@ -0,0 +1,133 @@
+import { Address, BigInt, ethereum } from "@graphprotocol/graph-ts";
+import { Account, LiquidityPool } from "../../generated/schema";
+import {
+  incrementPoolUniqueUsers,
+  incrementPoolUniqueDepositors,
+  incrementPoolUniqueBorrowers,
+  incrementPoolUniqueLiquidators,
+  incrementPoolUniqueLiquidatees,
+} from "./pool";
+import { EventType } from "./event";
+import {
+  BIGDECIMAL_ZERO,
+  BIGINT_ZERO,
+  INT_ONE,
+  INT_ZERO,
+  PositionSide,
+} from "../utils/constants";
+
+export function getOrCreateAccount(
+  event: ethereum.Event,
+  pool: LiquidityPool,
+  address: Address
+): Account {
+  let account = Account.load(address);
+  if (!account) {
+    account = new Account(address);
+    account.cumulativeEntryPremiumUSD = BIGDECIMAL_ZERO;
+    account.cumulativeExitPremiumUSD = BIGDECIMAL_ZERO;
+    account.cumulativeTotalPremiumUSD = BIGDECIMAL_ZERO;
+    account.cumulativeDepositPremiumUSD = BIGDECIMAL_ZERO;
+    account.cumulativeWithdrawPremiumUSD = BIGDECIMAL_ZERO;
+    account.cumulativeTotalLiquidityPremiumUSD = BIGDECIMAL_ZERO;
+
+    account.longPositionCount = INT_ZERO;
+    account.shortPositionCount = INT_ZERO;
+    account.openPositionCount = INT_ZERO;
+    account.closedPositionCount = INT_ZERO;
+    account.cumulativeUniqueLiquidatees = INT_ZERO;
+
+    account.depositCount = INT_ZERO;
+    account.withdrawCount = INT_ZERO;
+    account.borrowCount = INT_ZERO;
+    account.collateralInCount = INT_ZERO;
+    account.collateralOutCount = INT_ZERO;
+    account.liquidateCount = INT_ZERO;
+    account.liquidationCount = INT_ZERO;
+    account.swapCount = INT_ZERO;
+
+    account.save();
+
+    incrementPoolUniqueUsers(event, pool);
+  }
+  return account;
+}
+
+export function incrementAccountOpenPositionCount(
+  account: Account,
+  positionSide: string
+): void {
+  if (PositionSide.LONG == positionSide) {
+    account.longPositionCount += INT_ONE;
+  } else {
+    account.shortPositionCount += INT_ONE;
+  }
+  account.openPositionCount += INT_ONE;
+
+  account.save();
+}
+
+export function decrementAccountOpenPositionCount(
+  account: Account,
+  positionSide: string
+): void {
+  if (PositionSide.LONG == positionSide) {
+    account.longPositionCount -= INT_ONE;
+  } else {
+    account.shortPositionCount -= INT_ONE;
+  }
+  account.openPositionCount -= INT_ONE;
+  account.closedPositionCount += INT_ONE;
+
+  account.save();
+}
+
+export function incrementAccountEventCount(
+  event: ethereum.Event,
+  pool: LiquidityPool,
+  account: Account,
+  eventType: EventType,
+  sizeDelta: BigInt
```

Comment:
You don't seem to be using `sizeDelta` for anything?

Reply:
It is used at https://github.com/messari/subgraphs/blob/b68898ed16191780328266281c242cfe2f7e8ad7/subgraphs/mux-protocol/src/entities/account.ts#L104

Reply:
Oh! Missed that one, sorry 🙈